base_station/gui/src/components/ERDTask.vue

```diff
@@ -217,7 +221,8 @@ export default {
     OdometryReading,
     IKControls,
     PDBFuse,
-    DriveVelDataV
+    DriveVelDataV,
```

Comment:
just out of curiosity, what is DriveVelDataV?

Reply:
It's a component for displaying drive data, and the tables are arranged vertically. We also have DriveVelDataH that's wider rather than taller